package.json

```diff
@@ -480,9 +478,7 @@
         "subPermissions": [
           "patron-block-conditions.collection.get",
           "patron-block-conditions.item.get",
-          "patron-block-conditions.item.post",
-          "patron-block-conditions.item.put",
-          "patron-block-conditions.item.delete"
+          "patron-block-conditions.item.put"
```

Comment:
Without post and delete, the display name now feels incorrect. If this is correct, the `displayName` here and [the translation value](https://github.com/folio-org/ui-users/blob/5d5c67cdb46c7856ffc02fd0004b06b34e00e01c/translations/ui-users/en.json#L950) should be updated as well.

Reply:
Is there a corresponding `okapiInterfaces` change here too? I don't know which interface these permissions are associated with, but it would surprise me if it's `login`, the only change indicated here.

Reply:
Good question. They were removed here: https://github.com/folio-org/mod-patron-blocks/commit/849911db4d0b496347a1aa37a11c368e00469701

Reply:
I've updated the display name.

Reply:
I agree "Can view and edit..." is better. Please make this change [in the translation file]([the translation value](https://github.com/folio-org/ui-users/blob/5d5c67cdb46c7856ffc02fd0004b06b34e00e01c/translations/ui-users/en.json#L950) as well since that's the value that is actually displayed in the UI.